### PR TITLE
bugfix: deviceTwin membership msg unmarshal failed.

### LINKED ISSUE #890: Unmarshal update request body failed, Please check the request
When I enable the devicetwin,   the error messege from edge " Unmarshal update request body failed, Please check the request"

### PR DESCRIPTION
**Pull request by kexun**
**What type of PR is this?**
> /kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #890 

**Special notes for your reviewer**:

### CODE REVIEW

File: edge/pkg/devicetwin/dtcontroller.go [hunk 1]
```diff
@@ -300,42 +300,32 @@ func classifyMsg(message *dttype.DTMessage) bool {
 		log.LOGGER.Infof("Classify the msg to action %s", action)
 		return true
 	} else if (strings.Compare(msgSource, "edgemgr") == 0) || (strings.Compare(msgSource, "devicecontroller") == 0) {
-		if strings.Contains(message.Msg.Router.Resource, "membership/detail") {
-			message.Action = dtcommon.MemDetailResult
+		switch message.Msg.Content.(type) {
```
Comment (subpathdev) [inline edge/pkg/devicetwin/dtcontroller.go]: can you use an if else here?
With if else I think the readability would be increased.

Comment (sujithsimon22) [reply edge/pkg/devicetwin/dtcontroller.go]: +1

Comment (sujithsimon22) [reply edge/pkg/devicetwin/dtcontroller.go]: We could maybe use an assertion and if the assertion fails we can marshal the message ??

Comment (sids-b) [reply edge/pkg/devicetwin/dtcontroller.go]: I think it is better to keep the code style constant everywhere. We have a switch case in edged module

Comment (sujithsimon22) [reply edge/pkg/devicetwin/dtcontroller.go]: fair enough

**Pull request merged**